### PR TITLE
Prepare release 1.7.0

### PR DESCRIPTION
**Pull request by dbrgn**
Did we forget anything?

Things that need to be done before the release:

- [x] Small tweaks and improved documentation for "multiple platforms" feature: https://github.com/dbrgn/tealdeer/pull/300#issuecomment-1925882814
- [x] Bump dependencies:  #374

...anything else?

### CODE REVIEW

**Comment by niklasmohrin** [comment]
Yes I am still waiting for you in https://github.com/dbrgn/tealdeer/pull/345#issuecomment-2075993411 and would like to include that in the release

**Comment by dbrgn** [comment]
@niklasmohrin can you check your Matrix messages?

**Comment by dbrgn** [comment]
@niklasmohrin sorry for bothering you again over this channel, but I sent you another message 🙂

**Comment by niklasmohrin** [comment]
Hmmmm, apparently I have something misconfigured over there :thinking: :D

File: RELEASING.md [hunk 1]
```diff
@@ -7,16 +7,13 @@ Run linting:
 Set variables:
 
     $ export VERSION=X.Y.Z
-    $ export GPG_KEY=EA456E8BAF0109429583EED83578F667F2F3A5FA
+    $ export GPG_KEY=20EE002D778AE197EF7D0D2CB993FF98A90C9AB1
```
Comment (dbrgn) [inline RELEASING.md:10]: @niklasmohrin if you have a GPG signing key, you can replace this with your own key ID after the release.

**Comment by dbrgn** [comment]
@niklasmohrin I believe this is now ready. Could you check if everything looks good to you? Once that's done, I'll publish the release 🙂

**Comment by niklasmohrin** [comment]
Do we want to change the default features to `["native-tls"]`, or leave that for the next release?

**Comment by dbrgn** [comment]
Given that it's no urgent issue without workaround, and since it only affects people in networks that do active TLS interception, I think we can move that to the next release.

**Comment by niklasmohrin** [comment]
Alright then, I will aim for the next release to be earlier than two years anyways :D

**Review by niklasmohrin — APPROVED** [review]

**Pull request merged**